elm.json

```diff
@@ -8,11 +8,11 @@
         "direct": {
```

Comment:
I can't comment above this line but you should update your elm version to 0.19.1.